package.json

```diff
@@ -70,6 +71,8 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.438.0",
```

Comment:
Apparently these aws additions added 3.7k lines to package-lock -- turtle dependencies all the way down.

Reply:
😱